FrontPage/controllers/AddTaskViewController.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+class AddTaskViewController: UIViewController {
+  
+  
+  @IBOutlet weak var nameField: UITextField!
+  
+  @IBOutlet weak var descriptionField: UITextField!
+  
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    nameField.delegate = self
+    descriptionField.delegate = self
+  }
+  
+  override func didReceiveMemoryWarning() {
+    super.didReceiveMemoryWarning()
+  }
+  
+  @IBAction func addTask(_ sender: UIButton) {
+    
+    var taskStatus: TaskStatus
+    taskStatus = TaskStatus.open
+    
+    self.performSegue(withIdentifier: "PostListController", sender: self)
+    apollo.perform(mutation: CreateTaskMutation(title: nameField.text ?? "test1", description: descriptionField.text ?? "description of test1", status: taskStatus  )) { result in
```

Comment:
The default of t`test1` might be better as been `error`. If there is an error when creating the task a error message would be better than a message a user might enter.